Description: Clarify GPL license version.
Author: Stephen Leake <stephen_leake@stephen-leake.org>
Origin: upstream, commit: f79d8215061cd57a97ecfe3bd02a7748ff605cf5
Last-Update: 2014-05-07

--- a/src/unix/parse_date.cc
+++ b/src/unix/parse_date.cc
@@ -1,6 +1,6 @@
 // Copyright (C) 2010 Stephen Leake <stephen_leake@stephe-leake.org>
 //
-// This program is made available under the GNU GPL version 3.0 or
+// This program is made available under the GNU GPL version 2.0 or
 // greater. See the accompanying file COPYING for details.
 //
 // This program is distributed WITHOUT ANY WARRANTY; without even the
--- a/src/win32/parse_date.cc
+++ b/src/win32/parse_date.cc
@@ -1,6 +1,6 @@
 // Copyright (C) 2010 Stephen Leake <stephen_leake@stephe-leake.org>
 //
-// This program is made available under the GNU GPL version 3.0 or
+// This program is made available under the GNU GPL version 2.0 or
 // greater. See the accompanying file COPYING for details.
 //
 // This program is distributed WITHOUT ANY WARRANTY; without even the
